HtmlCssSample.css

```diff
@@ -0,0 +1,188 @@
+body {
+    background-color: rgb(35, 56, 76);
+    display: flex;
+    margin: 0;
+}
+
+nav {
+    position: fixed;
+    background-color: white;
+    height: 10%;
+    width: 100%;
+    display: flex;
+}
+
+.logo {
+    display: flex;
+    justify-content: flex-start;
+    margin-left: 2%;
+    margin-top: 1%;
+    width: 10%;
+    height: 50%;
+}
+
+img {
+    height: 100%;
+}
+
+div {
+    display: inline-flex;
+    justify-content: space-evenly;
+    width: 60%;
+    height: 100%;
+    padding-left: 40%;
+}
+
+p {
+    margin-top: 2%;
+    height: 30%;
+}
+
+p:hover {
+    background-color: rgb(235, 233, 233);
+    cursor: pointer;
+}
+
+.buttons {
+    margin-top: 2%;
+}
+
+.btn-sign-in {
+    border-radius: 5px;
+    height: 35%;
+}
+
+.btn-register {
+    border-radius: 5px;
+    height: 35%;
+    background-color: black;
+    color: white;
+}
+
+.btn-sign-in:hover {
+    cursor: pointer;
+}
+
+.btn-register:hover {
+    cursor: pointer;
+}
+
+.html-heading {
+    font-size: 300%;
```

Comment:
Use font-style instead of this kind of styling

Reply:
Resolved